procrustes/utils.py

```diff
@@ -262,8 +262,8 @@ def compute_error(a, b, t, s=None):
         \mathbf{A}\mathbf{T}`, and the reference array, :math:`\mathbf{B}`.
 
     """
-    # transform matrix A
-    a_trans = np.dot(a, t) if s is None else np.dot(np.dot(t.T, a), s)
+    # transform matrix A to either AT or SAT
+    a_trans = np.dot(a, t) if s is None else np.dot(np.dot(s, a), t)
```

Comment:
Just to clarify the purpose of this PR: This is the key change in this PR, and the rest of changes just adapt to this change in the `compute_error` function.

Reply:
Looks good to me.